go/api/base_client.go

```diff
@@ -165,6 +165,22 @@ func (client *baseClient) Get(key string) (string, error) {
 	return handleStringOrNullResponse(result), nil
 }
 
+func (client *baseClient) GetEx(key string) (string, error) {
+	result, err := client.executeCommand(C.GetEx, []string{key})
+	if err != nil {
+		return "", err
+	}
+	return handleStringOrNullResponse(result), nil
+}
+
+func (client *baseClient) GetExWithOptions(key string, options *GetExOptions) (string, error) {
+	result, err := client.executeCommand(C.GetEx, append([]string{key}, options.getExOptionsToArgs()...))
```

Comment:
Why did we have to rename toArgs() to toXXXArgs()?

Reply:
Did the renaming for better distinction, but on second thought, toArgs() is more readable. Changing it back.